recipes/pexports/all/patches/0003-fix-MSVC-x86.patch

```diff
@@ -0,0 +1,7 @@
+--- pexports.c
++++ pexports.c
+@@ -81,2 +81,2 @@
+-  extern __declspec(dllimport) void __stdcall *GetModuleHandleA(char *name);
+-  extern __declspec(dllimport) void __stdcall *GetProcAddress(void *module, char *name);
++  extern __declspec(dllimport) void * __stdcall GetModuleHandleA(char *name);
++  extern __declspec(dllimport) void * __stdcall GetProcAddress(void *module, char *name);
```

Comment:
for sure, these functions return pointers (HMODULE/HINSTANCE and a function pointer)
seems like upstream has a bug

Reply:
I think the C standard says to assume that EAX (=result variable) is clobbered.